sample.audius.conf

```diff
@@ -1,13 +1,30 @@
 # creator-node audius.conf
 CADDY_TLS='tls internal'
 creatorNodeEndpoint=https://localhost
-delegateOwnerWallet=0xb3c66e682Bf9a85F6800c769AC5A05c18C3F331d
-delegatePrivateKey=62ca4962f99a39b4a2c60a8402f60baea5437c275146abd90e0bfdcf0415184d
+delegateOwnerWallet=0x0D38e653eC28bdea5A2296fD5940aaB2D0B8875c
+delegatePrivateKey=21118f9a6de181061a2abd549511105adb4877cf9026f271092e6813b7cf58ab
```

Comment:
These are values from audius-compose land that work with the pre-deployed ganache container, not sure what the original values are from.